cmd/rook/ceph/osd.go

```diff
@@ -271,9 +271,9 @@ func commonOSDInit(cmd *cobra.Command) {
 // Parse the devices, which are comma separated. A colon indicates a non-default number of osds per device
 // or a non collocated metadata device.
 // For example, one osd will be created on each of sda and sdb, with 5 osds on the nvme01 device.
-//   sda:1,sdb:1,nvme01:5
+//   sda:1:auto,sdb:1:auto,nvme01:5:auto
 // For example, 3 osds will use sdb SSD for db and 3 osds will use sdc SSD for db.
-//   sdd:1:sdb,sde:1:sdb,sdf:1:sdb,sdg:1:sdc,sdh:1:sdc,sdi:1:sdc
+//   sdd:1:auto:sdb,sde:1:auto:sdb,sdf:1:auto:sdb,sdg:1:auto:sdc,sdh:1:auto:sdc,sdi:1:auto:sdc
```

Comment:
This string is getting complicated. This works for now, but next time a property is added we should consider a better way to serialize the settings.

Reply:
Agreed.